creator/ingest_runs/tasks/validation_run.py

```diff
@@ -1,79 +1,159 @@
 import os
+import json
+import jsonpickle
 import logging
+from pprint import pformat, pprint
+
+import pandas
 import django_rq
 from django.conf import settings
 from django.core.files.base import ContentFile
 from django_s3_storage.storage import S3Storage
 
+from kf_lib_data_ingest.etl.extract.utils import Extractor
+from kf_lib_data_ingest.validation.data_validator import (
+    Validator as DataValidator,
+)
+from kf_lib_data_ingest.validation.reporting.markdown import (
+    MarkdownReportBuilder,
+)
+
 from creator.decorators import task
 from creator.ingest_runs.models import ValidationRun, ValidationResultset
+from creator.files.models import extract_config_path, Version
+from creator.analyses.file_types import FILE_TYPES
+from creator.analyses.analyzer import extract_data
 
+ROOT_DIR = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+EXTRACT_CFG_DIR = os.path.join(ROOT_DIR, "extract_configs", "templates")
 S3_STORAGE = "django_s3_storage.storage.S3Storage"
 
 logger = logging.getLogger(__name__)
 
 
-def validate_file_versions(validation_run):
+def clean_and_map(version):
     """
-    TODO - Replace later. Currently counts to 10 then returns
-
-    Upload validation results and report files to storage backend
+    Load content of file version into a DataFrame and then use ingest lib's
+    extraction utility to clean the data and map the columns to the
+    standard set of columns expected by the validator
     """
-    logger.info(f"Begin validation for {validation_run.versions.all()}")
-    results = {}
+    logger.info(f"Performing extraction to clean and map file {version}")
+
+    # Load ingest extract operations for this file
+    if not version.root_file:
+        raise Exception(
+            f"Validation failed. Version {version} missing root_file"
+        )
+    ec = extract_config_path(version)
+    if not ec:
+        raise Exception(
```

Comment:
I'm not sure what the desired behavior is here. Is it possible to have a file in a data review that does not have a template? If yes, then I would change this to log a warning instead of raise an Exception